src/base_classes/active_space_solver.h

```diff
@@ -111,6 +112,23 @@ class ActiveSpaceSolver {
         as_ints_ = as_ints;
     }
 
+    /// Save the wave function
+    std::map<StateInfo, std::string> dump_wave_function();
+
+    /// Read the wave function from file as initial guess
+    void read_wave_function(const std::map<StateInfo, std::string>& state_filename_map) {
+        state_filename_map_ = state_filename_map;
+    };
+
+    /// Set energy convergence
```

Comment:
I am surprised you needed to add these. Is there another mechanism to set these variables?

Reply:
I am not sure what is the best way, but the `ActiveSpaceMethod` has them fixed in the [header](https://github.com/evangelistalab/forte/pull/227/files/11248643ecc15886d5e279b140bd25d65e44708c#diff-aa6bb529a4e6b980f3e07a088a6f95163d4ffcfb44da9f5cb8f6d00cca4da5deR208) and requires the children to override the `set_options` function. I add this simply because `ActiveSpaceSolver` is the user interface and during the MCSCF optimization, we probably don't need to converge things too tightly.